doc/changelog.rst

```diff
@@ -1,6 +1,39 @@
 Changelog
 =========
 
+Changes in Version 4.3
+----------------------
+
+PyMongo 4.3 brings a number of improvements including:
+
+- Added support for decoding BSON datetimes outside of the range supported
+  by Python's :class:`~datetime.datetime` See
+  :ref:`handling-out-of-range-datetimes` for examples, as well as
+  :class:`pymongo.datetime_ms.DatetimeMS`,
+  :attr:`pymongo.codec_options.CodecOptions.datetime_conversion`,
+  :class:`pymongo.datetime_ms.DatetimeConversion` for more details
```

Comment:
These 3 links don't render correctly: https://pymongo--1038.org.readthedocs.build/en/1038/changelog.html?highlight=datetime_#changes-in-version-4-3